pandas/core/algorithms.py

```diff
@@ -692,30 +692,34 @@ def factorize(values, sort=False, order=None, na_sentinel=-1, size_hint=None):
 
 
 def value_counts(
-    values, sort=True, ascending=False, normalize=False, bins=None, dropna=True
+    values,
+    sort: bool = True,
+    ascending: bool = False,
+    normalize: bool = False,
+    bins=None,
```

Comment:
```suggestion
    bins: Optional[bool] = None,
```

(unless docstring is wrong)

Reply:
docstring says int.  should it be bool?

Reply:
No this is a typo on my end

Reply:
Meant Optional[int] though? Can do in a follow up